int8 yaw_rate
int8 x_rate
int8 y_rate
---
bool result 
